build-system/tasks/karma.conf.js

```diff
@@ -26,18 +26,22 @@ module.exports = {
 
   preprocessors: {
     'src/**/*.js': ['browserify'],
+    'src/**/*.ts': ['browserify'],
```

Comment:
does this work if we consolidate it into the previous line?
```
'src/**/*.{js,ts}': ['browserify'],
```

Reply:
Yep, updated ✅